src/user-store.js

```diff
@@ -204,8 +204,13 @@ class UserStore {
 
   deleteUser (user) {
     let userKey = UserStore.normalizeIdKey(user.id)
-
-    return this.backend.del('users', userKey)
+    var deletedEmail = Promise.reject(new Error('No email given'))
```

Comment:
This would halter the removal of the accountDir in `node-solid-server/lib/requests/delete-account-confirm-request.js`, right? This would make it impossible for users with no email to delete their account. Which might be what we want, but it ties into my question at https://github.com/solid/node-solid-server/issues/855#issuecomment-428722553

Reply:
Indeed, it would, but since they couldn't get a token, that's should be OK. I suppose the alternative is that the email delete resolves also if there isn't an email set... To the current way of deleting an account through NSS, it doesn't matter, but I don't know what should be the behavior here...

Reply:
If we're not allowing deleting unless you have a email address, we could change the logic to:

```
if (!user.email) {
  return Promise.reject(new Error('No email given'))
}
// do rest of the stuff
```

No need to do `backend.del` unless we have valid email, right?